README.md

```diff
@@ -1 +1,72 @@
-# Student-Portfolio-2
\ No newline at end of file
+# Project Name
+
+> Student-Portfolio
+
+
+## Built With
+
+- Major languages
+HTML and CSS
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link]()
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
```

Comment:
- [ ] Kindly customize the README template to reflect what you have exactly done in this milestone.